lib/realmapi.js

```diff
@@ -48,15 +48,27 @@ function realmAPI(path, opts, extraopts, callback) {
 		ifr.src = url
 		return
 	}
-
+/*
 	queue_request({
 		dataType: 'jsonp',
 		url: 'https://query.yahooapis.com/v1/public/yql',
 		data: {
 			q: 'select * from xml where url="' + url + '"',
 			format: 'json'
 		},
-		complete: callback
+		complete: function(result){
+            console.log(JSON.stringify(result));
+            callback(result);
+        }
+	})
+*/
+	queue_request({
+		url: 'https://cors-anywhere.herokuapp.com/' + url,
+		complete: function(result){
+            var jsonResult = x2js.xml_str2json(result.responseText);
+            //console.log(JSON.stringify(jsonResult));
+            callback(jsonResult);
```

Comment:
you changed the type of the argument that is passed to callback (it used to be the xhr object) and didn't fix all instances where this function is used (namely the `migrate/progress` call in `mule.js`)